Bond/Bond+Arrays.swift

```diff
@@ -82,6 +86,12 @@ public class DynamicArray<T>: Dynamic<Array<T>>, SequenceType {
     bond.bind(self, fire: fire, strongly: strongly)
   }
   
+  public override var value: Array<T> {
+    didSet {
+      dynCount.value = count
+    }
+  }
+  
```

Comment:
What is the policy about directly modifying `DynamicArray.value`? Is that something that should be supported at all?